src/boot/ws.js

```diff
@@ -104,17 +104,21 @@ onWebSocket(dbEnum.wsCategory.init, (data) =>
   console.log(`Init message received: ${data}`)
 )
 
-onWebSocket(dbEnum.wsCategory.tick, (data) =>
-  console.log(`Tick received: ${data}`)
-)
```

Comment:
Can we tie these to some debug flag so we automatically get these when we turn on the debug level. I remeber Timotej did something like this. Same goes for all the other commented out code in this file

Reply:
We have a good logging on back-end tied to loglevels. I don't even know how this is done on front-end.

Would it make sense for back-end to control this? 

Or does it make more sense to this fully in front-end?

Reply:
BTW: This is a cleanup. We should not hold up the PR for this.

Reply:
My main concern was that if that was added by someone for some error monitoring. So I was wondering if we will need that again. May be have the front end show this for now. We could come up with a way to show all issue in one place later?